frontend/templates/index.html

```diff
@@ -52,7 +52,7 @@
 </head>
 <body>
   <div class="pdf-container">
-    <embed src="{{ url_for('get_pdf', name='dummy.pdf') }}" type="application/pdf" width="100%" height="100%" />
+    <embed src="{{ url_for('get_pdf', pdf_id=dummy_pdf_id) }}" type="application/pdf" width="100%" height="100%" />
```

Comment:
These changes look great, thanks! If you could edit this one thing:
```suggestion
    <embed src="{{ url_for('pdf_object_operations', pdf_id=dummy_pdf_id) }}" type="application/pdf" width="100%" height="100%" />
```

The index page pops up an error currently because it's still using the old endpoint.

Reply:
Thanks for pointing that out!